config/config.go

```diff
@@ -100,6 +100,8 @@ type Configuration struct {
 	Hooks       Hooks       `mapstructure:"hooks"`
 	Validations Validations `mapstructure:"validations"`
 	PriceFloors PriceFloors `mapstructure:"price_floors"`
+
+	Auction Auction `mapstructure:"auction"`
```

Comment:
Please remove. This is already covered by `AccountDefaults`.

Reply:
done